mp3spi/pom.xml

```diff
@@ -50,6 +50,7 @@
                 <configuration>
                     <instructions>
                         <SPI-Provider>javax.sound.sampled.AudioSystem</SPI-Provider>
+                        <Export-Package>javazoom.spi</Export-Package>

```

Comment:
I don't understand the need for changing this - currently released version already export the package - `javazoom.spi;version="1.9.5"`